src/components/home/chat-bubble/components/__tests__/image-message.test.tsx

```
@@ -0,0 +1,105 @@
+import { render, screen } from "@testing-library/react";
+import { vi } from "vitest";
+import { ImageMessage } from "../image-message";
+import { IMessage } from "../../../../types";
+
+// Mock Next.js Image component
+vi.mock("next/image", () => ({
+  default: ({ src, alt, onClick, className, fill }: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)